CONTRIBUTING.md

```diff
@@ -164,6 +172,10 @@ For example:
 - Deploy OSS on GKE, run  `./bin/start --oss --gke`
 - Deploy DAP on Openshift, run  `./bin/start --dap --oc311`
 
+It is also possible to run a single test instead of the full suite. This can be done by running `./bin/start` with the
+flag `--test-prefix=<prefix>`. For example, running with the flag `--test-prefix=TEST_ID_18` will run only the test
```

Comment:
This is indeed a great improvement!
Suggestion: Give more examples, such as `--test-prefix=*1[789]` to run tests 17,18,19